server/chat-plugins/chatlog.ts

```diff
@@ -636,8 +678,9 @@ export const commands: ChatCommands = {
 	searchlog: 'searchlogs',
 	searchlogs(target, room) {
 		target = target.trim();
-		const [search, tarRoom, limit, date] = target.split(',');
+		const [search, tarRoom, limit, date] = target.split(',').map(str => str.trim());
 		if (!target) return this.parse('/help searchlogs');
+		if (search.length < 3) return this.errorReply(`Too short of a search query.`);
```

Comment:
This should fall back to showing command help, incidentally.

Reply:
Limit should also be optional; why is it suddenly required?

Reply:
room should also be optional; it should default to the current room...

Reply:
limit is required since it will always be passed, since there's now a max cap of 3000 to prevent OOM

Reply:
Oh, this is the length of the search text! My bad.

Reply:
This should definitely specify what the minimum is. This is a very confusing error message.

Reply:
I don't think this is necessary, anyway. Short search queries should probably be exact-word matches, but there's nothing wrong with that. As long as the limit exists, this doesn't need to also exist.

Reply:
I benchmarked FS search with this limit several times. It strains it a lot - in the zone of like, 20 second searches, which worries me when you consider that likely people will be like "it's not working? better try again." Additionally, I don't see a benefit to being able to search one or two letters. It would just lag, and you'd get a _lot_ of false positives. 
It seems like, since this is already here, why not keep it and keep the extra lag protection?

Reply:
FS search should definitely show a loading screen if it takes that long; as should everything else that takes that long.

As I mentioned on Discord:

I think there are certain letter combinations like `XD` where you won't get many false positives; it seems way worse to refuse to give good results than to sometimes not give bad results.

I'm also confused why it would strain over this search but not other searches. The default limit is 500, right? It should be _faster_ than a search with 0 results, because it wouldn't need to search anywhere near as much of the history.